pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -1336,12 +1337,17 @@ impl RemoteTimelineClient {
                     // In-place replace of Initialized to Stopped can be done with the help of https://github.com/Sgeo/take_mut
                     // but for this use case it doesnt really makes sense to bring unsafe code only for this usage point.
                     // Deletion is not really perf sensitive so there shouldnt be any problems with cloning a fraction of it.
```

Comment:
Lets just drop this comment.

Reply:
I think this comment is still accurate for the moment?  We are still copying the queue when going from Initialized to Stopped (later, this should go away when we simplify deletion in https://github.com/neondatabase/neon/issues/5080)

Reply:
I just want to avoid adding aborting paths in crates (which take_mut has to do), but it was not really relevant for this PR. Resolving.